data/part-4/2-objects-in-a-list.md

```diff
@@ -93,13 +96,16 @@ while (index < names.size()) {
     index = index + 1;
 }
 
+System.out.println();
+
 // 2. for loop with index
 for (int i = 0; i < names.size(); i++) {
     System.out.println(names.get(i));
 }
 
 System.out.println();
-// 3. for each loop (no index)
+
+// 3. for-each loop (no index)
 for (String name: names) {
```

Comment:
should also be consistent with (using) for-each loop syntax, so:

`for (String name : names) {`
...